lms/static/js/spec/staff_debug_actions_spec.js

```diff
@@ -11,134 +11,158 @@ define([
         describe('StaffDebugActions', function() {
             var location = 'i4x://edX/Open_DemoX/edx_demo_course/problem/test_loc';
             var locationName = 'test_loc';
-            var fixture_id = 'sd_fu_' + locationName;
-            var fixture = $('<input>', {id: fixture_id, placeholder: 'userman'});
+            var fixtureID = 'sd_fu_' + locationName;
+            var $fixture = $('<input>', {id: fixtureID, placeholder: 'userman'});
             var escapableLocationName = 'test\.\*\+\?\^\:\$\{\}\(\)\|\]\[loc';
-            var escapableFixture_id = 'sd_fu_' + escapableLocationName;
-            var escapableFixture = $('<input>', {id: escapableFixture_id, placeholder: 'userman'});
+            var escapableFixtureID = 'sd_fu_' + escapableLocationName;
+            var $escapableFixture = $('<input>', {id: escapableFixtureID, placeholder: 'userman'});
             var esclocationName = 'P2:problem_1';
             var escapableId = 'result_' + esclocationName;
             var escapableResultArea = $('<div>', {id: escapableId});
 
-            describe('get_url ', function() {
+            describe('getURL ', function() {
                 it('defines url to courseware ajax entry point', function() {
-                    spyOn(StaffDebug, 'get_current_url')
+                    spyOn(StaffDebug, 'getCurrentUrl')
                       .and.returnValue('/courses/edX/Open_DemoX/edx_demo_course/courseware/stuff');
-                    expect(StaffDebug.get_url('rescore_problem'))
+                    expect(StaffDebug.getURL('rescore_problem'))
                       .toBe('/courses/edX/Open_DemoX/edx_demo_course/instructor/api/rescore_problem');
                 });
             });
 
-            describe('sanitize_string', function() {
+            describe('sanitizedString', function() {
```

Comment:
This changed from `sanitize_string` to `sanitizedString`, with a "d"; was that intentional?